packages/react-scripts/package.json

```diff
@@ -60,6 +61,9 @@
     "recursive-readdir": "2.1.0",
     "strip-ansi": "3.0.1",
     "style-loader": "0.13.1",
+    "stylelint": "7.6.0",
+    "stylelint-config-standard": "15.0.0",
+    "stylelint-webpack-plugin": "0.4.1",
```

Comment:
@karlhorky I recommend another update here to capture our fixes from this morning.